docs/concepts/operator.md

```diff
@@ -232,6 +232,106 @@ Please refer to [this document](../usage/shoot_credentials_rotation.md#gardener-
+
+### `virtual-garden-kube-apiserver` Exposure
+
+The `virtual-garden-kube-apiserver` is exposed via a dedicated `istio-ingressgateway` deployed to namespace `virtual-garden-istio-ingress`.
+The `virtual-garden-kube-apiserver` `Service` in the `garden` namespace is only of type `ClusterIP`.
+Consequently, DNS records for this API server must target the load balancer IP of the `istio-ingressgateway`.
+
+### Virtual Garden Kubeconfig
+
+`gardener-operator` does not generate any static token or likewise for access to the virtual cluster.
+Ideally, human users access it via OIDC only.
+Alternatively, you can create an auto-rotated token that you can use for automation like CI/CD pipelines:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: shoot-access-virtual-garden
+  namespace: garden
+  labels:
+    resources.gardener.cloud/purpose: token-requestor
+  annotations:
+    serviceaccount.resources.gardener.cloud/name: virtual-garden-user
+    serviceaccount.resources.gardener.cloud/namespace: kube-system
+    serviceaccount.resources.gardener.cloud/token-expiration-duration: 3h
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: managedresource-virtual-garden-access
+  namespace: garden
+type: Opaque
+stringData:
+  clusterrolebinding____gardener.cloud.virtual-garden-access.yaml: |
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: gardener.cloud.sap:virtual-garden
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cluster-admin
+    subjects:
+    - kind: ServiceAccount
+      name: virtual-garden-user
+      namespace: kube-system
+---
+apiVersion: resources.gardener.cloud/v1alpha1
+kind: ManagedResource
+metadata:
+  name: virtual-garden-access
+  namespace: garden
+spec:
+  secretRefs:
+  - name: managedresource-virtual-garden-access
+```
+
+The `shoot-access-virtual-garden` `Secret` will get a `.data.token` field which can be used to authenticate against the virtual garden cluster.
+See also [this document](resource-manager.md#tokenrequestor-controller) for more information about the `TokenRequestor`.
+
+### `gardener-apiserver`
+
+Similar to the [`virtual-garden-kube-apiserver`](#virtual-garden-kube-apiserver), the `gardener-apiserver` also uses a few certificates and other credentials that should not change during the upgrade.
+Again, you have to prepare the environment accordingly by leveraging the [secret's manager capabilities](../development/secrets_management.md#migrating-existing-secrets-to-secretsmanager).
+
+- The existing ETCD Encryption Key `Secret` should be labeled with `secrets-manager-use-data-for-name=gardener-apiserver-etcd-encryption-key`.
+
+> Please test your upgrade procedure thoroughly.
+> Note that in some cases it can be easier to set up a fresh landscape with `gardener-operator`, restore the ETCD data, switch the DNS records, and issue new credentials for all clients.
```

Comment:
This seems good advise that should be generally followed, but it looks like it is just part of the `gardener-apiserver` section. Maybe it is a better idea to move it into the top section.
WDYT?